### PR TITLE
Add back button to appbar and button section

### PR DESCRIPTION
**Pull request by korneliakobiela**
[Issue] #1188
[Problem] [UiComponents] App doesn't have back button in every place.
[Solution] - Add backbutton to sections Appbar and Button

Signed-off-by: Kornelia Kobiela <korneliak.95@gmail.com>

### CODE REVIEW

**Review by HunseopJeong — APPROVED** [review]
LGTM

File: examples/mobile/UIComponents/components/buttons/contained/index.html [hunk 1]
```diff
@@ -11,6 +11,9 @@
 
 <body>
 	<div class="ui-page" id="ContainedButtonsDemo">
+		<div class="ui-appbar-left-icons-container">
+			<a href="#" class="ui-btn ui-btn-icon ui-btn-icon-back" data-style="flat" data-rel="back"></a>
+		</div>
```
Comment (lslachciak) [inline examples/mobile/UIComponents/components/buttons/contained/index.html]: please double check code here, view is broken in this example on my PC
image

File: examples/mobile/UIComponents/components/appbar/multiline/multiline-title.html [hunk 2]
```diff
@@ -12,6 +12,9 @@
 <body>
 	<div class="ui-page" id="multilineTitle">
 		<header>
+			<div class="ui-appbar-left-icons-container">
```
Comment (lslachciak) [inline examples/mobile/UIComponents/components/appbar/multiline/multiline-title.html:15]: we need to decide and be consistent if we add back button to submenus or examples too.
This is example not submenu.
If we add back button to examples we will need to add to all of them
e.g. also
examples/mobile/UIComponents/components/etc/units/units.html
examples/mobile/UIComponents/components/etc/units/fonts.html
examples/mobile/UIComponents/components/etc/interactive/coverflow/index.html
examples/mobile/UIComponents/components/etc/interactive/interactive3D/index.html
examples/mobile/UIComponents/components/etc/about/index.html

File: examples/mobile/UIComponents/components/appbar/multiline/subtitle.html [hunk 3]
```diff
@@ -12,6 +12,9 @@
 <body>
 	<div class="ui-page" id="subtitle">
 		<header>
```
Comment (lslachciak) [inline examples/mobile/UIComponents/components/appbar/multiline/subtitle.html:14]: ditto.

File: examples/mobile/UIComponents/components/buttons/contained/body-area/2-buttons.html [hunk 4]
```diff
@@ -11,6 +11,12 @@
 
 <body>
 	<div class="ui-page" id="ContainedButtonsBodyArea2Buttons">
+		<header>
```
Comment (lslachciak) [inline examples/mobile/UIComponents/components/buttons/contained/body-area/2-buttons.html:14]: ditto.

**Review by lslachciak — CHANGES_REQUESTED** [review]
Please check my comments in code,
also I see missing back button in following submenus
examples/mobile/UIComponents/components/toast/index.html
examples/mobile/UIComponents/components/textfields/index.html
examples/mobile/UIComponents/components/etc/index.html
examples/mobile/UIComponents/components/etc/interactive/index.html
examples/mobile/UIComponents/components/etc/graphs/index.html

**Review by lslachciak — APPROVED** [review]
seems good.
Toast, Checkbox and Etc main menu will need back button in other PR

**Pull request merged**